charts/airflow/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -241,15 +241,15 @@ spec:
               {{- end }}
               {{- if .Values.scheduler.variables }}
                && echo "*** adding Airflow variables..." \
-               && airflow variables -i /home/airflow/variables-pools/variables.json \
+               && airflow variables -i /home/airflow/variables-pools/variables.json || airflow variables import /home/airflow/variables-pools/variables.json \
```

Comment:
I am not very confortable with this or, do you do this to maintain backward compatibility? I guess we can drop support for old airflow version (this would require at least a minor chart bump)

Reply:
Its mostly for backward compatibility cause the moment we switch to 2.0+ it breaks. If the decision is to stop supporting previous versions (which also makes sense since 2.0 is inherently different in setup), then that is not necessary